migrations/20200516011910-initial-data-load.js

```diff
@@ -0,0 +1,312 @@
+        novelID: 1,
+        genreID: 7
+      }, {
+        novelID: 1,
+        genreID: 10
+      }, {
+        novelID: 2,
+        genreID: 7
+      }, {
+        novelID: 2,
+        genreID: 15
+      }, {
+        novelID: 2,
+        genreID: 5
+      }, {
+        novelID: 3,
+        genreID: 7
+      }, {
+        novelID: 3,
+        genreID: 14
+      }, {
+        novelID: 3,
+        genreID: 12
+      }, {
+        novelID: 4,
+        genreID: 7
+      }, {
+        novelID: 4,
+        genreID: 11
+      }, {
+        novelID: 4,
+        genreID: 6
+      }, {
+        novelID: 5,
+        genreID: 7
+      }, {
+        novelID: 5,
+        genreID: 15
+      }, {
+        novelID: 5,
+        genreID: 5
+      }, {
+        novelID: 6,
+        genreID: 7
+      }, {
+        novelID: 6,
+        genreID: 12
+      }, {
+        novelID: 7,
+        genreID: 7
+      }, {
+        novelID: 7,
+        genreID: 11
+      }, {
+        novelID: 8,
+        genreID: 7
+      }, {
+        novelID: 8,
+        genreID: 4
+      }, {
+        novelID: 8,
+        genreID: 10
+      }, {
+        novelID: 8,
+        genreID: 13
+      }, {
+        novelID: 9,
+        genreID: 7
+      }, {
+        novelID: 9,
+        genreID: 3
+      }, {
+        novelID: 9,
+        genreID: 12
+      }, {
+        novelID: 9,
+        genreID: 16
+      }, {
+        novelID: 10,
+        genreID: 7
+      }, {
+        novelID: 10,
+        genreID: 11
+      }, {
+        novelID: 10,
+        genreID: 9
+      }, {
+        novelID: 10,
+        genreID: 6
+      }, {
+        novelID: 11,
+        genreID: 7
+      }, {
+        novelID: 11,
+        genreID: 12
+      }, {
+        novelID: 11,
+        genreID: 15
+      }, {
+        novelID: 11,
+        genreID: 11
+      }, {
+        novelID: 12,
+        genreID: 7
+      }, {
+        novelID: 12,
+        genreID: 8
+      }, {
+        novelID: 12,
+        genreID: 10
+      }, {
+        novelID: 12,
+        genreID: 1
+      }, {
+        novelID: 13,
+        genreID: 7
+      }, {
+        novelID: 13,
+        genreID: 1
+      }, {
+        novelID: 13,
+        genreID: 17
+      }, {
+        novelID: 14,
+        genreID: 7
+      }, {
+        novelID: 14,
+        genreID: 2
+      }, {
+        novelID: 14,
+        genreID: 10
+      }, {
+        novelID: 15,
+        genreID: 7
+      }, {
+        novelID: 15,
+        genreID: 10
+      }, {
+        novelID: 15,
+        genreID: 18
+      }, {
+        novelID: 15,
+        genreID: 14
+      }
+    ])
+  },
+
+  down: (queryInterface) => {
+    /*
+      Add reverting commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.dropTable('users');
+    */
+    return queryInterface.bulkDelete('authors'),
+      queryInterface.bulkDelete('genres'),
+      queryInterface.bulkDelete('novels'),
+      queryInterface.bulkDelete('novelGenres')
```

Comment:
Without the awaiting here all of these run at the same time meaning it could attempt to delete from `novelGenres` before delete from `novels` which would cause it to fail.

```JavaScript
await queryInterface.bulkDelete('authors')
await queryInterface.bulkDelete('genres')
await queryInterface.bulkDelete('novels')

return queryInterface.bulkDelete('novelGenres')
```